api_v3/lib/types/EntryServerNode/live/KalturaLiveEntryServerNode.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+class KalturaLiveEntryServerNode extends KalturaEntryServerNode {
```

Comment:
I see no reason to have array of parameters.
Please the parameters directly on the object.